app/views/admin/shared/_global_var_ofn.js.erb

```diff
@@ -0,0 +1 @@
+var ofn_<%= name %> = <%= value.to_json.html_safe %>;
```

Comment:
I'm surprised. I was imagining one global variable containing all our config:
```suggestion
var OFN = {
  "<%= name %>": <%= value.to_json.html_safe %>
}
```

But it doesn't really matter. :shrug:

Reply:
I thought the same, then I thought I should be defensive and not clobber an existing var (in case the partial is used a second time for another config):
```suggestion
var ofn = ofn || {};
ofn['<%= name %>'] = <%= value.to_json.html_safe %>;
```

But then I though it would be better to register all configs into a global variable and print out once, and then.. I went back to the minimum needed ;)